packages/babel-plugin-strip-milestones/README.md

```diff
@@ -0,0 +1,9 @@
+# @milestones/babel-plugin-strip-milestones
+
+This package allows you to strip the `milestone()` calls completely out of your code to eliminate any overhead in production.
+
+Full documentation can be found at https://salsify.github.io/milestones/docs/babel-plugin.
+
+## Usage
+
+Include `@milestones/babel-plugin-strip-milestones` in your package dependencies and then add `'@milestones/strip-milestones'` to the `plugins` array in your `babel.config.json` or the corresponding configuration location for your particular Babel integration.
```

Comment:
Just want to make sure this is right... The package is `@milestones/babel-plugin-strip-milestones` but the plugin name is `@milestones/strip-milestones`?

Reply:
Yep, Babel has some [shorthand rules](https://babeljs.io/docs/en/plugins#plugin-shorthand) for specifying plugins. I should probably just spell the full thing out to avoid confusion, though.